app/views/admin/people/show.html.erb

```diff
@@ -11,7 +11,7 @@
 
     <article class="person">
 
-      <% if @person.image %>
+      <% if @person.image_url %>
```

Comment:
Is it likely there'll be other cases like this? Might be something we need to check carefully.